### PR TITLE
chore: small fixes to multicontainer samples

### PR DESCRIPTION
**Pull request by shubha-rajan**

### CODE REVIEW

File: examples/multi-container/ruby/README.md [hunk 1]
```diff
@@ -98,6 +98,8 @@ spec:
 
              # Ensure the port number on the --port argument matches the value of the DB_PORT env var on the my-app container.
              - "--port=5432"
+            # Instance URIs follow the format 
+            # projects/PROJECT_ID/locations/REGION_ID/clusters/CLUSTER_ID/instances/INSTANCE_ID
```
Comment (jackwotherspoon) [inline examples/multi-container/ruby/README.md:102]: would `REGION_ID`, `CLUSTER_ID`, and `INSTANCE_ID` be better off as `REGION_NAME`, `CLUSTER_NAME`, and `INSTANCE_NAME`. If these are referred to as IDs in the docs then we can leave as is.

File: examples/multi-container/ruby/multicontainers.yaml
Comment (jackwotherspoon) [inline examples/multi-container/ruby/multicontainers.yaml:1]: let's name this file `multicontainer.yaml`

File: examples/multi-container/ruby/README.md [hunk 2]
```diff
@@ -59,8 +59,8 @@ DB = connect_tcp()
 gcloud builds submit --tag gcr.io/<YOUR_PROJECT_ID>/run-alloydb
 ```
 
-Finally, create a revision YAML file (multicontainers.yaml), using the `example.yaml`
-file as a referece for the deployment, listing the AlloyDB container image as a sidecar:
+Finally, update the `multicontainers.yaml` file with the correct values for your
```
Comment (jackwotherspoon) [inline examples/multi-container/ruby/README.md:62]: to be consistent with Cloud SQL example
```suggestion
Finally, update the `multicontainer.yaml` file with the correct values for your
```

**Review by jackwotherspoon — CHANGES_REQUESTED** [review]

**Comment by jackwotherspoon** [comment]
@shubha-rajan friendly ping here

File: examples/multi-container/ruby/README.md (hunk 1 above)
Comment (shubha-rajan) [reply examples/multi-container/ruby/README.md:102]: I copied this straight from the docs!

**Review by jackwotherspoon — APPROVED** [review]

**Pull request merged**